src/oge/gross_to_net_generation.py

```diff
@@ -193,19 +183,19 @@ def calculate_gross_to_net_conversion_factors(
             on=["plant_id_eia", "subplant_id", "report_date"],
             validate="1:1",
         )
-        .merge(
-            subplant_capacity,
-            how="left",
-            on=["plant_id_eia", "subplant_id"],
-            validate="m:1",
-        )
         .merge(
             net_gen_data,
             how="outer",
             on=["plant_id_eia", "subplant_id", "report_date"],
             indicator="data_source",
             validate="1:1",
         )
+        .merge(
```

Comment:
Moved this to after merging in the EIA data so that prime mover codes would be added for all generators.